recipes-appfw/sample-node0/files/manifest.in

```diff
@@ -0,0 +1,21 @@
+{
+    "application": "node0",
+    "description": "node0 - A NodeJS sample app.",
+    "provider": "test",
+    "groups": "test-node0",
+    "environment": {
+        "NODE0_FOO": "node0-foo",
+        "NODE0_BAR": "node0-bar",
+    },
+    "command": [
```

Comment:
Hmm... Good idea, I think that should work as long as there is no default.target in /etc on the host side as /etc has higher priority than /run.

Actually, an alternative would be to generate at bitbake time an application overlay-specific /etc/systemd/system/default.target[.wants] and populate it with symlinks to the services we want to run inside the container. Since it is in /etc, that would have the highest priority and since it would be overlaid on top of the host /etc/systemd/system/default.target it would also suppress it.

Well, have to try in practice to see if these ideas work.